sdlib/d/gc/arena.d

```diff
@@ -203,7 +203,7 @@ package:
 	}
 
 	void collect(ref CachedExtentMap emap, ubyte gcCycle) shared {
-		filler.collect(emap, gcCycle);
+		filler.collect(emap, gcCycle, containsPointers);
```

Comment:
Why do we need to pass containsPointers around everywhere here? The arena knows if it contains pointers or not. The PageFiller know what Arena it is in.

Reply:
> The PageFiller know what Arena it is in

I didn't realize this, will fix.